### PR TITLE
Challenges component on Home Page

### PR DESCRIPTION
**Pull request by ShantanuLamba**

### CODE REVIEW

File: frontend/src/pages/toplevel/home/Challenges.vue [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      <center>
+        <h1 class="pb-2" style="font-size: 3.5rem">C h a l l e n g e s</h1>
+      </center>
+      <h2 class="mt-4" style="">Browse...</h2>
+      <p style="letter-spacing: 1.4px">
+        Choose a challenge, read the problem, write code, submit , that's it!
+      </p>
+      <div>
+        <b-row>
+          <b-col><button class="button-random mt-2">RANDOM</button></b-col>
```
Comment (aman2712) [inline frontend/src/pages/toplevel/home/Challenges.vue:13]: the correct way to use this button is:
`<b-button variant="primary" block class="mt-2"> </b-button>`

File: frontend/src/pages/toplevel/home/Challenges.vue [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        { ID:3 , Problem: "Example", Languages: "Example" ,Difficulty:"Medium"},
+        { ID:4 , Problem: "Example", Languages: "Example" ,Difficulty:"Medium"},
+        { ID:5 , Problem: "Example", Languages: "Example" ,Difficulty:"Medium"},
+      ],
+    };
+  },
+};
+</script>
+<style scoped>
+.button-random {
```
Comment (aman2712) [inline frontend/src/pages/toplevel/home/Challenges.vue:66]: You don't need this class as I mentioned in my last comment

**Review by aman2712 — DISMISSED** [review]
This is just one change I found, you can fix it and it might be ready for production

File: frontend/src/pages/toplevel/home/Challenges.vue (hunk 2 above)
Comment (ShantanuLamba) [reply frontend/src/pages/toplevel/home/Challenges.vue:66]: Thanks I will do it

File: frontend/src/pages/toplevel/home/Challenges.vue [hunk 3]
```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="bg-dark my-5">
+    <b-container class="text-light">
+      <center>
```
Comment (Wittmaxi) [inline frontend/src/pages/toplevel/home/Challenges.vue:4]: Use class `text-center` on parent or (for block-level elements with fixed width) `mx-auto`

File: frontend/src/pages/toplevel/home/Challenges.vue [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        <h1 class="pb-2" style="font-size: 3.5rem">C h a l l e n g e s</h1>
```
Comment (Wittmaxi) [inline frontend/src/pages/toplevel/home/Challenges.vue:5]: Please stick to the bootstrap-fontsizes to guarantee consistency. If you need a bigger font-size, use `display-<1-4>` (defined by bootstrap)

File: frontend/src/pages/toplevel/home/Challenges.vue [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+      </center>
+      <h2 class="mt-4" style="">Browse...</h2>
+      <p style="letter-spacing: 1.4px">
```
Comment (Wittmaxi) [inline frontend/src/pages/toplevel/home/Challenges.vue:8]: Please stick to the default letter-spacing to guarantee consistency

File: frontend/src/pages/toplevel/home/Challenges.vue [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+              <b-dropdown-item>First Action</b-dropdown-item>
+              <b-dropdown-item>Second Action</b-dropdown-item>
+              <b-dropdown-item>Third Action</b-dropdown-item>
+            </b-dropdown>
+          </b-col>
+        </b-row>
+      </div>
+
+      <div>
+        <center>
```
Comment (Wittmaxi) [inline frontend/src/pages/toplevel/home/Challenges.vue:40]: There is no need to center a block-level element

File: frontend/src/pages/toplevel/home/Challenges.vue [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+  <div class="bg-dark my-5">
+    <b-container class="text-light">
+      <center>
+        <h1 class="pb-2" style="font-size: 3.5rem">C h a l l e n g e s</h1>
+      </center>
+      <h2 class="mt-4" style="">Browse...</h2>
+      <p style="letter-spacing: 1.4px">
+        Choose a challenge, read the problem, write code, submit , that's it!
+      </p>
+      <div>
```
Comment (Wittmaxi) [inline frontend/src/pages/toplevel/home/Challenges.vue:11]: Those div's have no effect and only clutter the file

File: frontend/src/pages/toplevel/home/Challenges.vue [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+            <b-dropdown id="dropdown-1" text="Sort By" class="m-md-2 mt-1">
+              <b-dropdown-item>First Action</b-dropdown-item>
+              <b-dropdown-item>Second Action</b-dropdown-item>
+              <b-dropdown-item>Third Action</b-dropdown-item>
+            </b-dropdown>
+          </b-col>
+        </b-row>
+      </div>
+
+      <div>
```
Comment (Wittmaxi) [inline frontend/src/pages/toplevel/home/Challenges.vue:39]: those divs have no effect and only clutter the file

**Review by Wittmaxi — CHANGES_REQUESTED** [review]
Far better than your last PR! Just some very small details that you couldn't have known since we never talked about them as they are rather specific!

File: frontend/src/pages/toplevel/home/Challenges.vue (hunk 4 above)
Comment (ShantanuLamba) [reply frontend/src/pages/toplevel/home/Challenges.vue:5]: is that a class?

Comment (Wittmaxi) [reply frontend/src/pages/toplevel/home/Challenges.vue:5]: https://getbootstrap.com/docs/4.0/utilities/display/

**Pull request merged**